### PR TITLE
Add integration tests for the new WAYF

### PR DESCRIPTION
**Pull request by Badlapje**
Added preliminary integration tests for the WAYF screen in the new design.
Fixed them as far as i could for now (without interactivity) and commented on those which need to be fixed in the future.  Skipped those that can't work atm.

### CODE REVIEW

File: theme/cypress/integration/skeune/wayf/UnconnectedIdpPicker.spec.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import {UnconnectedIdpPicker} from "../../../../base/javascripts/modules/UnconnectedIdpPicker";
+
+it('Check if the UnconnectedIdpPicker constructor was called', () => {
+    const unconnectedIdpPicker = new UnconnectedIdpPicker(null, null, null);
+
+    expect(unconnectedIdpPicker).instanceof(UnconnectedIdpPicker);
+});
```
Comment (MKodde) [inline theme/cypress/integration/skeune/wayf/UnconnectedIdpPicker.spec.js:7]: This feels like code duplication to me. This code is already tested in the 'openconext' theme. That should be enough don't you think? 

What would be even better is to run the unit tests in another 'suite' which can be theme independent. Not something to fix here and now..

I suggest removing it from `skeune` for now

File: theme/cypress/integration/skeune/wayf/WayfMouseBehaviour.spec.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+context('WayfMouseBehaviour', () => {
```
Comment (MKodde) [inline theme/cypress/integration/skeune/wayf/WayfMouseBehaviour.spec.js:1]: I feel this test is also very `openconext` theme related. This test covers a bugfix that is specific for that theme. When we are changing behaviour of WAYF, this test probably becomes obsolete. 

I suggest removing it from `skeune`

**Review by MKodde — COMMENTED** [review]
Other than the two requests for removal, this looks good to go!

File: theme/cypress/integration/skeune/wayf/WayfMouseBehaviour.spec.js (hunk 2 above)
Comment (Badlapje) [reply theme/cypress/integration/skeune/wayf/WayfMouseBehaviour.spec.js:1]: ah, makes sense to me if it's openconext specific :-)

File: theme/cypress/integration/skeune/wayf/UnconnectedIdpPicker.spec.js (hunk 1 above)
Comment (Badlapje) [reply theme/cypress/integration/skeune/wayf/UnconnectedIdpPicker.spec.js]: Removed it for now, but added a task in the leftover ticket for this.

Even though it's already tested in the openconext theme, changes in the skeune theme might break it.  So it's better to test it for every theme.  It is code duplication though, so fully agree with that point.  We shall find a way ;-)

**Review by MKodde — APPROVED** [review]

**Pull request merged**